src/components/controls/mouse-controls.js

```diff
@@ -0,0 +1,157 @@
+var registerControls = require('../../core/controls').registerControls;
+var THREE = require('../../lib/three');
+var debug = require('../../utils/debug');
+
+var warn = debug('components:mouse-controls:warn');
+
+/**
+ * Mouse-controls component.
+ *
+ * Updates rotation with mousedrag or pointerlock input.
+ */
+module.exports.Component = registerControls('mouse-controls', {
+  schema: {
+    enabled: { default: true },
+    pointerlockEnabled: { default: false },
```

Comment:
I'm starting to think Pointer Lock controls should be a separate third-party controls component. We're getting more serious about actual VR, and rotating the camera with Pointer Lock is not something you'd do in VR.

The downside is responsiveness with flat screens. But I think we'll see responsiveness in VR means that it works on multiple headsets/platforms, not that it works on everything. Though if someone wants to use A-Frame for flat experiences, they are free to pull/implement a custom third-party Pointer Lock controls.

So we should maybe just get rid of Pointer Lock from the core repo and publish it as an easy-to-consume third-party repo. It increases testing overhead and code maintenance by a fair amount. @dmarcos Agree?